components/Conversation.tsx

```diff
@@ -1,17 +1,40 @@
 import React from 'react';
 import styled from 'styled-components';
 import { flexCentered } from './styles';
+import Participant from './Participant';
 
 const ConversationContainer = styled.div`
   grid-row: top / bottom;
   grid-column: left / right;
   ${flexCentered};
 `;
 
-const Conversation = () => {
+const Conversation = ({ localParticipant, participants }) => {
+  const isYou = participant => {
+    return localParticipant.identity === participant.identity;
+  };
```

Comment:
Prettier is fine with it, but because the line is long, it turns into this, spread over 2 lines without braces:

```js
const isYou = participant =>
  localParticipant.identity === participant.identity;
```

What do you think?

Reply:
IMO if it has to drop down (it looks to be one character too long), we should use curlies. That's easier to immediately parse if it ends up being multi-line regardless.